btree/btree.c

```diff
@@ -3342,16 +3370,19 @@ static int64_t btree_put_makespace_phase(struct m0_btree_op *bop)
 	temp_rec_1.r_key.k_data   = M0_BUFVEC_INIT_BUF(&p_key_1, &ksize_1);
 	temp_rec_1.r_val          = M0_BUFVEC_INIT_BUF(&p_val_1, &vsize_1);
 
-	for (i = oi->i_used - 1; i >= 0; i--) {
-		node_put(&oi->i_nop, lev->l_alloc, true, bop->bo_tx);
-		lev->l_alloc = NULL;
+	node_put(&oi->i_nop, lev->l_alloc, true, bop->bo_tx);
```

Comment:
Can move this to line 3358, this will look more coherent.

Reply:
No, I also wanted to move at line 3358, but we need l_alloc to make new record, ref. line 3366.